lib/debuglet.js

```diff
@@ -215,7 +217,21 @@ Debuglet.prototype.scheduleBreakpointFetch_ = function(seconds) {
 
         default:
           that.logger_.info('\t' + response.statusCode + ' completed.');
-          that.updateActiveBreakpoints_(body.breakpoints || []);
+          var bps = (body.breakpoints || []).filter(function(bp) {
+            if (bp.action !== 'CAPTURE') {
+              that.logger_.warn('Found breakpoint with invalid action:', bp.action);
+              bp.status = new StatusMessage(StatusMessage.BREAKPOINT_ACTION,
+                BREAKPOINT_ACTION_MESSAGE, true);
+              that.debugletApi_.updateBreakpoint(bp, function(err/*, body*/) {
```

Comment:
`completeBreakpoint_` updates our internal data structures for tracking breakpoints and removes the breakpoint from the v8 debug object. Breakpoints with invalid actions can be updated without ever being entered into the breakpoint map or being reported to V8. We could alternatively provide a separate path through these functions for breakpoints that were never registered but that felt like unnecessary complexity.